src/core/Akka.API.Tests/verify/CoreAPISpec.ApproveClusterTools.Core.verified.txt

```diff
@@ -353,9 +353,8 @@ namespace Akka.Cluster.Tools.Singleton
     }
     public sealed class ClusterSingletonManagerSettings : Akka.Actor.INoSerializationVerificationNeeded
     {
-        public ClusterSingletonManagerSettings(string singletonName, string role, System.TimeSpan removalMargin, System.TimeSpan handOverRetryInterval, bool considerAppVersion) { }
```

Comment:
Do we have to keep these old API signatures in order to preserve binary compatibility @Arkatufus ?

Reply:
we do, I think I missed it